tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
+  sourceMaps: "inline",
+  compact: false,
+  whitelist: [
```

Comment:
Is there any way to avoid having this list in two places?  They are already different.